common/src/main/webapp/less/style.less

```diff
@@ -1089,19 +1089,24 @@ tfoot {
 @mobile-responsive-cutoff: 670px;
 
 .target-form {
-  grid-template-columns: [label] minmax(100px, 1fr) [field] 6fr;
+  grid-template-columns: [label] minmax(100px, 10%) [field] 1fr;
   justify-items: stretch;
 }
 
 .target-search-form {
   grid-column: 1 / span 2;
   display: grid;
-  grid-template-columns: [label] minmax(100px, 1fr) [field] 6fr;
+  grid-template-columns: [label] minmax(100px, 10%) [field] minmax(calc(100% - 100px - 0.3em), calc(90% - 0.3em));
   grid-gap: 0.3em;
 
   .field {
     margin-bottom: 0 !important;
   }
+
+  .input {
```

Comment:
Is this the `input` class from SUI? Should we specify that in the selector as `.ui.input` ?

Reply:
Right. I wonder if we can avoid the `important`

Reply:
Yes, it is the `input` class from SUI, that's why I have modified it inside the `target-search-form` class, to avoid breaking anything else. If you think it can be a breaking change a new class can be created for that specific component or use `.ui.input` as @rpiaggio mentioned.
Also the `important` can be omitted in the `display: grid`.
I will apply both suggestions.